ui/lib/client/translations/en.yaml

```diff
@@ -3,6 +3,7 @@ error:
   network: Network connection error
   api:
     unauthorized: Session is expired. Please sign in again.
+    invalid_request: Bad Request
```

Comment:
When will this happen?

Reply:
no data can be exported

Reply:
Then can we make error message better for it? IMO "Invalid request" can't let users understand what happened.

Reply:
Due to the current implementation of error handling in FE, it is not suitable to provide special handling for this PR. I think it is necessary to create a separate PR to optimize the error handling.